src/Explorer/Explorer.ts

```diff
@@ -1839,6 +1848,20 @@ export default class Explorer {
     return false;
   }
 
+  public setSelfServeType(inputs: ViewModels.DataExplorerInputsFrame): void {

```

Comment:
Can you add this without touching Explorer.ts at all? Move it all up into the React layer that loads and instantiates Explorer.ts. I would very much like to avoid any new KO->React adapters or adding more tech debt that couples us to `Explorer.ts`

Reply:
For this, I ll need to change Main.tsx from a react function to react class (since i need to reference explorer to set/change states). Thats a wholesale change right, and out of scope for this PR?